x-pack/plugins/apm/public/components/app/settings/agent_explorer/agent_list/index.tsx

```diff
@@ -153,6 +157,43 @@ export function getAgentsColumns({
         />
       ),
     },
+    {
+      field: AgentExplorerFieldName.AgentLastVersion,
+      name: (
+        <EuiToolTip
+          content={i18n.translate(
+            'xpack.apm.agentExplorerTable.agentLatestVersionColumnTooltip',
+            {
+              defaultMessage: 'The latest released version of the agent.',
+            }
+          )}
+        >
+          <>
+            {i18n.translate(
+              'xpack.apm.agentExplorerTable.agentLatestVersionColumnLabel',
+              { defaultMessage: 'Agent Latest Version' }
```

Comment:
What about "Latest agent version"?

Reply:
Addressed in [3130a75](https://github.com/elastic/kibana/pull/153643/commits/3130a7506f9572fe9a633ffaeeacbf7c9b9937aa).